exporter/exporterhelper/common.go

```diff
@@ -213,3 +271,22 @@ func (ts *timeoutSender) send(req internal.Request) error {
 	}
 	return req.Export(ctx)
 }
+
+func createSampledLogger(logger *zap.Logger) *zap.Logger {
```

Comment:
We need to revisit this, seems very arbitrary to have a sampler logger only for some cases. Maybe just better document as comment of the func why we have this and where should be used, but it is still random for me.

Reply:
I agree. We have another effort to address this https://github.com/open-telemetry/opentelemetry-collector/pull/8134. I'm just copying it from one file to another here

Reply:
Thank you @dmitryax ,
Right as part of #8134 we want to have a Sampled logger as part of the TelemetrySetting and use it when needed. 

Please @bogdandrutu @dmitryax take a look at my PR #8134. Thanks :)